examples/data-sources/vmds_certificates/data_source_certificates.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    vmds = {
+      source = "hashicorp.com/svc-bot-mds/vmds"
+    }
+  }
+}
+
+provider "vmds" {
+  host      = "https://tdh-cp-vh.tdh.kr.com"
```

Comment:
please remove the valid username password and url in example files. Replace it with dummy data